website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,473 @@
+| Developer 1 | dev           | dbt_dev1      |dev.dbt_dev1_marketing.my_model            |
+| Developer 2 | dev           | dbt_dev2      |dev.dbt_dev2_marketing.my_model            |
+| CI PR 123   | ci            | dbt_pr_123    |ci.ci_schema.dbt_pr_123_marketing_my_model |
+| CI PR 234   | ci            | dbt_pr_234    |ci.ci_schema.dbt_pr_234_marketing_my_model |
+| Production  | prod          | analytics     |prod.marketing.my_model                    |
+
+
+### What NOT to do
+
+
+### Update generate_schema_name() to always use the custom schema if it is set
+
+
+Some people prefer to only use the custom schema when it is set instead of concatenating the default schema with the custom one, as it happens in the out of the box behavior.
+
+
+#### Problem
+
+When modifying the default macro for `generate_schema_name()`, this might result iney might creatinge this new version.
+
+```jinja2
+
+{% macro generate_schema_name(custom_schema_name, node) -%}
+
+    {%- set default_schema = target.schema -%}
+    {%- if custom_schema_name is none -%}
+
+        {{ default_schema }}
+
+    {%- else -%}
+    # The following is incorrect as it omits {{ default_schema }} before {{ custom_schema_name | trim }}. 
+        {{ custom_schema_name | trim }} 
+
+    {%- endif -%}
+
+{%- endmacro %}
+```
+
+While it may provide the expected output for Production, where a dedicated database is used, it will generate conflicts anywhere people share a database. 
+
+
+Let’s look at the example of a model called `my_model` with a custom schema of `marketing`.
+
+
+| Context     |Target database| Target schema | Resulting object               |
+|-------------|:-------------:|:-------------:|:------------------------------:|
+| Production  | prod          | analytics     |prod.marketing.my_model         |
+| Developer 1 | dev           | dbt_dev1      |dev.marketing.my_model          |
+| Developer 2 | dev           | dbt_dev2      |dev.marketing.my_model          |
+| CI PR 123   | ci            | dbt_pr_123    |ci.marketing.my_model           |
+| CI PR 234   | ci            | dbt_pr_234    |ci.marketing.my_model           |
+
+
+We can see that both Developer 1 and Developer 2 get the same object for `my_model`. This means that if they both work on this model at the same time, it will be impossible to know if the version currently in the data warehouse is the one from Developer 1 and Developer 2.
+
+Similarly, different PRs will result in the exact same object in the data warehouse. If different PRs are open at the same time and modifying the same models, it is very likely that we will get issues, slowing down the whole development and code promotion.
+
+
+### Solution
+
+As described above in the first example, the macro should be updated to check if we are currently running dbt in Production or not. It is only when in Production that we want to remove the concatenation and only use the custom schema.
```

Comment:
sugg removing above as we dont use spatial location words for accessibility reasons:

```suggestion
As described in the first example, the macro should be updated to check if we are currently running dbt in production or not. It's only when in production that we want to remove the concatenation and only use the custom schema.
```